packages/hamburger-react/src/Hamburger.tsx

```diff
@@ -20,18 +22,24 @@ export const Hamburger = ({
     onClick,
     inverted = false,
     description = "Hovedmeny",
-    className = "",
+    className,
     actionLabel,
 }: Props) => {
     const componentClassname = classnames(
         "jkl-hamburger",
         {
             "jkl-hamburger--is-open": isOpen,
             "jkl-hamburger--inverted": inverted,
+            "jkl-hamburger--label-before": actionLabel && actionLabel.position !== "after",
```

Comment:
Kan man gjøre det sånn ? 

"jkl-hamburger--label-before":  actionLabel?.position !== "after"

Reply:
Ja, det burde fungere 👍

Reply:
Hmm.. Dette blir nok litt feil. Og jeg ser at jeg også introduserte en feil her. Greia er at man vil automatisk ha `--label-after` dersom `actionLabel` finnes, så lenge `actionLabel.position` _ikke_ er `before`.

For at det skal funke helt som forventet må det være `actionLabel?.position === "before"` her, og `actionLabel && actionLabel.position !== "before"` på neste linje.. Jeg lager et forslag.